src/main/java/am/ik/yavi/core/ViolationMessage.java

```diff
@@ -115,7 +115,12 @@ public String messageKey() {
 				"\"{1}\" is/are not allowed for \"{0}\""), //
 		PASSWORD_REQUIRED("password.required", "\"{0}\" must meet {1} policy"), //
 		PASSWORD_OPTIONAL("password.optional",
-				"\"{0}\" must meet at least {1} policies from {2}") //
+				"\"{0}\" must meet at least {1} policies from {2}"), //
+		DATE_BEFORE("date.before", "\"{0}\" has to be before {1}"), //
```

Comment:
Isn't it inconsistent that the package name is `time` but the message key prefix is `date`?

How about unifying with `datetime`?

Reply:
Using `datetime` would be quite misleading since `LocalDate` does not store time.
I could rename it to `temporal`

Reply:
It sounds good

Reply:
Its already pushed